qa/common/gen_qa_trt_plugin_models.py

```diff
@@ -272,13 +273,37 @@ def create_plugin_models(models_dir):
     )
 
 
+def windows_load_plugin_lib(win_plugin_dll):
+    if os.path.isfile(win_plugin_dll):
+        try:
+            ctypes.CDLL(win_plugin_dll, winmode=0)
+        except TypeError:
+            # winmode only introduced in python 3.8
+            ctypes.CDLL(win_plugin_dll)
+        return
+
+    raise IOError('Failed to load library: "{}".'.format(win_plugin_dll))
+
+
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument(
         "--models_dir", type=str, required=True, help="Top-level model directory"
     )
+    parser.add_argument(
+        "--win_plugin_dll",
```

Comment:
@fpetrini15 do we need this flag? 
In fact discovery can be narrowed to the os/platform verification isn't?